report_wkhtmltopdf_param/security/ir.model.access.csv

```diff
@@ -1,3 +1,4 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "paperformat_parameter_access_portal","ir_actions_report_paperformat_parameter group_portal","model_report_paperformat_parameter","base.group_portal",1,0,0,0
 "paperformat_parameter_access_employee","ir_actions_report_paperformat_parameter group_hr_user","model_report_paperformat_parameter",,1,0,1,0
+"paperformat_parameter_access_administration","ir_actions_report_paperformat_parameter group_system","model_report_paperformat_parameter","base.group_system",1,0,1,1
```

Comment:
@Abranes why no write rights here?

Reply:
@gdgellatly I forgot, now that group can do everything.